### ISSUE TITLE
Trouble installing Cheat Engine

### ISSUE THREAD

**Issue by User-322516** [cheat-engine/cheat-engine#2963]
I'm already a user of Cheat Engine but when I have my new system (AMD Ryzen 7 5700 and RTX 6600), I can't install it anymore. When I try to install it, this happens. I tried installing this in a clean windows installation, still having the same problems. I don't have this kind of problem on my intel laptop. Is there a fix? Please help. 

![Annotation 2024-07-27 160748](https://github.com/user-attachments/assets/d4eaf566-4eb3-4cd8-8b12-f7e2242b9834)

I also tried compiling the source code using Lazarus but when I try to open cheat tables, well it also ends up in error. I've also tested other versions and the same thing happens. I've tested 7.4 and 7.3 versions.

**Comment by jpaulcipriano11** [comment]
I have the same problem. any fix?

**Comment by fade2gray** [comment]
If you have an issue installing CE on one device but not another, if both devices are connecting to the internet using the same router, then it could be a firewall issue on the device that you can't install to.

Otherwise, try using a command prompt to ping `d1vdn3r1396bak.cloudfront.net`. If you're unable to ping it, you may need to use a VPN

Another option is, if your PC has WiFi, you could try setting up a mobile hotspot with your mobile device and use that to download and install.

Ref: https://github.com/cheat-engine/cheat-engine/issues/2899#issuecomment-2254061090

**Comment by User-322516** [comment]
I can ping it just fine. I just don't get why this happens. Is there other solution? I've also tried setting up my mobile hotspot to install it, but the problem still persists. I've tried also using VPN but still the same problem. Please help. 

![image](https://github.com/user-attachments/assets/828c1024-d953-4b11-8821-ba23d6d57388)

**Comment by User-322516** [comment]
This is also the traceroute. Is this normal where there are request time out?

![image](https://github.com/user-attachments/assets/70463666-c787-4902-a143-3d176b6a2085)

**Comment by User-322516** [comment]
This is also the traceroute. Is this normal because there request time out?

![Screenshot 2024-07-29 113512](https://github.com/user-attachments/assets/9262dfa9-f443-45de-afa8-bef04dba3465)

**Comment by fade2gray** [comment]
I just noticed you are tying to install a Polish version - have you tried installing the English version?

**Comment by User-322516** [comment]
Yes. I've tried the English ver. from the main page. I also downloaded the other language from mirror links found in the site. I've tested and used other versions as well.

**Comment by fade2gray** [comment]
Are you certain Windows Defender, or any 3rd party security software, isn't blocking it - have you tried temporarily adding your `C:\Users\username\AppData\Local\Temp` folder as an excluded folder during installation?

As a last resort, you could just copy the CE installation folder from your old system to your new system.
